### PR TITLE
fix rafter upgrade

### PR DESCRIPTION
**Pull request by m00g3n**
**Description**

Changes proposed in this pull request:

- Fix rafter minio upgrade job

**Related issue(s)**

### CODE REVIEW

File: resources/rafter/charts/controller-manager/charts/minio/templates/secrets.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
-{{- if not .Values.existingSecret }}
+{{- if (eq (len (lookup "v1" "Secret" .Release.Namespace (include "minio.fullname" .))) 0) }}
```
Comment (m00g3n) [inline resources/rafter/charts/controller-manager/charts/minio/templates/secrets.yaml:1]: handle existing secret case

File: resources/rafter/charts/controller-manager/charts/minio/templates/_helper_patch_credentials.txt [hunk 2]
```diff
@@ -4,45 +4,17 @@ set -o errexit
...
+readonly NAMESPACE=$1
+readonly SECRET_NAME=$2
+readonly ACCESS_KEY=$3
+readonly SECRET_KEY=$4
+
+# if the secret with the given name is found in the given namespace,
+# annotate it with helm.sh/resource-policy: keep annotation;
+# otherwise create the secret
+
+( kubectl get secret -n ${NAMESPACE} ${SECRET_NAME} \
```
Comment (m00g3n) [inline resources/rafter/charts/controller-manager/charts/minio/templates/_helper_patch_credentials.txt]: handle existing secret case

File: resources/rafter/charts/controller-manager/charts/minio/templates/deployment.yaml [hunk 3]
```diff
@@ -137,12 +136,12 @@ spec:
             - name: MINIO_ACCESS_KEY
               valueFrom:
                 secretKeyRef:
-                  name: {{ if .Values.existingSecret }}{{ .Values.existingSecret }}{{ else }}{{ template "minio.fullname" . }}{{ end }}
```
Comment (m00g3n) [inline resources/rafter/charts/controller-manager/charts/minio/templates/deployment.yaml:140]: revert this

**Review by tgorgol — APPROVED** [review]

**Review by kfurgol — APPROVED** [review]

**Pull request merged**